models/users.js

```diff
@@ -42,9 +42,17 @@ const addOrUpdate = async (userData, userId = null) => {
       const isNewUser = !user.data();
       // user exists update user
       if (user.data()) {
+        // remove id field if exist in fetched data or profileDiff
+        const currentData = user.data();
```

Comment:
Does this has id in it?

Reply:
Will this ever get id? according to our model

Reply:
This PR is only for tests. Will be updating this after merging that and then only tests would be present and then would be merging.

Reply:
Then do a rebase of it?

Reply:
Yes